<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FM_SPEC_via_DG3000</class>
 <widget class="QMainWindow" name="FM_SPEC_via_DG3000">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FM_SPEC_via_DG3000</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="channel1">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>20</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>channel: 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="channel2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>channel: 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ch1_active">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>70</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="waveform">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>110</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>waveform</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_ch1_wform">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>110</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SIN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RAMP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SQU</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="box_ch2_wform">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>110</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SIN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RAMP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SQU</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="amplitude">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>220</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>amplitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="frequency">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>150</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="offset">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>290</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>offset</string>
    </property>
   </widget>
   <widget class="QLabel" name="phase">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>360</y>
      <width>34</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>phase</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="hscrollbar_ch1_freq">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>176</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2000000000</number>
    </property>
    <property name="pageStep">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDial" name="dial_ch1_phase">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchesVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="textbox_ch1_freq">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>150</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="textbox_ch1_phase">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>360</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_quit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>500</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>quit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textbox_ch1_ampl">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>220</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="textbox_ch1_off">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>290</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="active">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>70</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>active</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ch2_active">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>70</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_get">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>get setting</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="hscrollbar_ch1_ampl">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>246</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="hscrollbar_ch1_off">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>316</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="hscrollbar_ch2_ampl">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>246</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="hscrollbar_ch2_off">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>316</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="textbox_ch2_off">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>290</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="textbox_ch2_freq">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>150</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollBar" name="hscrollbar_ch2_freq">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>176</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000000000</number>
    </property>
    <property name="pageStep">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="textbox_ch2_ampl">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>220</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDial" name="dial_ch2_phase">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>390</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="notchesVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="textbox_ch2_phase">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>[ Hz ]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>[ Vpp ]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>[ V ]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>[ deg ]</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
